samples/Mvc.AjaxAntiforgeryAuth.Web/.gitignore

```diff
@@ -0,0 +1 @@
+wwwroot/lib/
```

Comment:
nothing changed. I was just tired of seeing a bunch of files installed from bower as pending changes in git.